src/ceph-volume/ceph_volume/devices/lvm/zap.py

```diff
@@ -49,6 +49,9 @@ def __init__(self, argv):
     @decorators.needs_root
     def zap(self, args):
         device = args.device
+        if disk.is_mapper_device(device):
+            terminal.error("Refusing to zap the mapper device: {}".format(device))
+            return
```

Comment:
for third-party systems like ceph-ansible, it is useful to return a non-zero exit status. It might be a good idea to `raise SystemExit(1)` here or have an exception with the terminal.error message (it gets handled automatically)